llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -861,8 +871,8 @@ int64_t HWAddressSanitizer::getAccessInfo(bool IsWrite,
 }
 
 HWAddressSanitizer::ShadowTagCheckInfo
-HWAddressSanitizer::insertShadowTagCheck(Value *Ptr,
-                                         Instruction *InsertBefore) {
+HWAddressSanitizer::insertShadowTagCheck(Value *Ptr, Instruction *InsertBefore,
+                                         DomTreeUpdater *DTU, LoopInfo *LI) {
```

Comment:
Across all those interfaces, both `DTU` and `LI` will always be non-null. Better to pass as reference?

Reply:
Good idea, done!